packages/containers/src/List/List.container.js

```diff
@@ -259,15 +264,24 @@ class List extends React.Component {
 				props.toolbar.sort.isDescending = !state.sortAsc;
 				props.toolbar.sort.field = state.sortOn;
 				props.toolbar.sort.onChange = (event, data) => {
+					if (props.toolbar.sort.onChangeSortOrder) {
+						props.toolbar.sort.onChangeSortOrder(event, data);
+					}
 					this.onSelectSortBy(event, data);
 				};
 			}
 
 			if (props.toolbar.filter) {
 				props.toolbar.filter.onToggle = (event, data) => {
+					if (props.toolbar.filter.onToggleFilter) {
+						props.toolbar.filter.onToggleFilter();
```

Comment:
Is there a reason for not sending the `event` and `data` ?

Reply:
no specific reason for excluding the attributes, its just that we are not using those values. However I agree that we might need these info in some other scenarios so Its added now.